Zotero/Scenes/Master/MasterContainerViewController.swift

```diff
@@ -43,9 +43,9 @@ final class MasterContainerViewController: UINavigationController {
     lazy var bottomController: DraggableViewController? = {
         return coordinatorDelegate?.createBottomController()
     }()
-    private var bottomContainer: UIView?
-    private var bottomYConstraint: NSLayoutConstraint!
-    private var bottomContainerBottomConstraint: NSLayoutConstraint!
+    private weak var bottomContainer: UIView?
+    private weak var bottomYConstraint: NSLayoutConstraint?
+    private weak var bottomContainerBottomConstraint: NSLayoutConstraint?
```

Comment:
Why make them weak? Just in case they are removed e.g. from the view controller's view, to be deallocated then? Is there a cyclic reference somwehere?